packages/SwingSet/src/typeGuards.js

```diff
@@ -1,12 +1,17 @@
 // @jessie-check
 import { M } from '@agoric/store';
 
-export const ManagerType = M.or('xsnap', 'xs-worker', 'local');
+export const ManagerType = M.or(
+  'xsnap',
+  'xs-worker',
+  'node-subprocess',
+  'local',
+);
 
 const Bundle = M.splitRecord({ moduleType: M.string() });
 
 const SwingsetConfigOptions = harden({
-  creationOptions: M.splitRecord({}, { critial: M.boolean() }),
+  creationOptions: M.splitRecord({}, { critical: M.boolean() }),
```

Comment:
huh, do we not have any test that set the `critical` flag on a static vat? does that test not use the typechecker? oh, we might have all unit test bypass the typechecker.

but the code in `initializeSwingset` that references this should be typechecked..

@erights you know thie typeguards way better than us.. what do you think?

Reply:
We have tests that test the `critical` flag, but nothing in SwingSet actually makes use of the `SwingSetConfigOptions` type guard or any of the type guards that descend from it.